### PR TITLE
Add auth request event

### PR DESCRIPTION
**Pull request by jan-law**
Related #461 

Not tested yet.
How can I trigger the `OpenShiftAuthManager` to emit an event?

### CODE REVIEW

**Comment by andrewazores** [comment]
> How can I trigger the OpenShiftAuthManager to emit an event?

You would need to build your Cryostat image and push it to a registry (ex. quay.io , under your own account/namespace). Then you can deploy your custom Cryostat image in CRC using the Operator. Once it's running in CRC it will automatically select the OpenShiftAuthManager, so set up a recording with the ALL metatemplate and after that most things you do will trigger the event to be recorded.

**Review by andrewazores — DISMISSED** [review]
I've just built and deployed this to my crc instance and it looks great.

**Comment by andrewazores** [comment]
[authrequest.tar.gz](https://github.com/cryostatio/cryostat/files/6716990/authrequest.tar.gz)

**Comment by jan-law** [comment]
Glad it's working! I actually haven't been able to test this yet because my `crc start` still isn't working. Guillaume from the crc team has been helping me with the issue

File: src/main/extras/app/Cryostat.xml [hunk 1]
```diff
@@ -3,13 +3,24 @@
      see Window -> Flight Recorder Template Manager.
 -->
 <configuration version="2.0" label="Cryostat" description="Continuous self-profiling for Cryostat" provider="Cryostat">
- <event name="io.cryostat.net.TargetConnectionManager.JMXConnectionOpened">
+ <event name="io.cyostat.net.OpenShiftAuthManager.AuthRequest">
```
Comment (andrewazores) [inline src/main/extras/app/Cryostat.xml:6]: typo: 'cyostat'

**Review by andrewazores — CHANGES_REQUESTED** [review]
The `web-client` submodule shouldn't be updated by this PR

**Pull request merged**